config.js.default

```diff
@@ -29,6 +29,11 @@ const config = {
     "Closet" : {autoOff : true}
   },
 
+  pirSensor : {
+    button : 12,
+    motion : 18
```

Comment:
You forgot this one ;)

Reply:
sorry, i didn't get you. i renamed the key to `pirSensor` . is there anything else i forgot ?

Reply:
Yes, but the default config still says motion : 18

Reply:
Ahha, I ment you should change 'motion : 18', not the parent json